### ISSUE TITLE
Slow down as log file grows larger

### ISSUE THREAD

**Issue by deadair3210** [gibaBR/Switch-Backup-Manager#146]
It seems that the program slows down as the log file grows larger due to the program writing to the file after every nsp, is there a way to maybe have it write to the log in chunks or maybe set a limit to the logfile's size?

**Comment by gibaBR** [comment]
Yes, I know about that problem. Didn't had time to solve it, but if you hit
the clear log button, it works as a workaround.

Em seg, 19 de ago de 2019 14:06, deadair3210 <notifications@github.com>
escreveu:

**Comment by deadair3210** [comment]
I do, i also just set the logfile as readonly for long jobs that I dont need the log of. Just wanted to ask to make sure there wasnt a better way
